### PR TITLE
Issue #7172 fixed

### PR DESCRIPTION
**Pull request by vijaysingh-webkul**
## Issue Reference
#7172

### CODE REVIEW

**Comment by Sanjaybhattwebkul** [comment]
@vijaysingh-webkul 
Resolve the **conflicts**.

**Comment by Sanjaybhattwebkul** [comment]
@vijaysingh-webkul 
Still getting the **conflicts**.

**Comment by Sanjaybhattwebkul** [comment]
@vijaysingh-webkul 
**Not working** as **expected resul**t. please take a **reference** from the [attached screenshot](https://github.com/bagisto/bagisto/issues/7172)

File: packages/Webkul/Velocity/src/Resources/assets/js/UI/components/product-collections.vue [hunk 1]
```diff
@@ -78,7 +77,9 @@
                 windowWidth: window.innerWidth,
             }
         },
-
+        updated() {
+            this.getProductSlideBlurEffect()
+        },
```
Comment (devansh-webkul) [inline packages/Webkul/Velocity/src/Resources/assets/js/UI/components/product-collections.vue:82]: Add a new line before and after the end of the function.

**Review by devansh-webkul — CHANGES_REQUESTED** [review]

**Pull request closed without merging**